PProPanel/jsx/general.jsx

```diff
@@ -0,0 +1,6 @@
+if (typeof $ == 'undefined') {
```

Comment:
Is this actually ever the case for ExtendScript contexts? I was under the impression that $ was a built-in global (see: Object Model Viewer or page 216 of the [JavaScript Tools Guide](http://wwwimages.adobe.com/content/dam/Adobe/en/devnet/scripting/pdfs/javascript_tools_guide.pdf)).